packages/react/cypress/integration/index-test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axe from '../../dist/index';
```

Comment:
Since this is now in a monorepo, we should be able to do this:
```suggestion
import axe from '@axe-core/react';
```